src/utils/fetch-twitch-videos.js

```diff
@@ -0,0 +1,22 @@
+import { get } from './request';
+
+const TWITCH_API_ENDPOINT = 'https://api.twitch.tv/helix/';
+const CLIENT_ID = '041r2glmgub2pt357ss0la44j2sz95';
+const MDB_CHANNEL_ID = '467752938';
+const video_URL = `${TWITCH_API_ENDPOINT}videos?user_id=${MDB_CHANNEL_ID}&first=`;
+const headers = { 'Client-ID': CLIENT_ID };
+
+// Fetches videos from twitch api
+const fetchTwitchVideos = async (videoLimit = 1) => {
```

Comment:
It would be great to use this in the `use-twitch-api` hook to abstract out some small details (such as having to get `.data` on the response, etc.). Perhaps in the `use-twitch-api` hook we can do something like the following with this:

```
            if (!currentStream && videoLimit) {
                const videoResp = await fetchTwitchVideos(some video limit);
                // return the whole array of videos, but ignore pagination for now
                setVideos(videoResp);
            }
```